packit/cli/scan_in_osh.py

```diff
@@ -0,0 +1,82 @@
+# Copyright Contributors to the Packit project.
+# SPDX-License-Identifier: MIT
+
+import json
+import logging
+import os
+import sys
+
+import click
+
+from packit.cli.types import LocalProjectParameter
+from packit.cli.utils import cover_packit_exception, get_packit_api, iterate_packages
+from packit.config import (
+    get_context_settings,
+    pass_config,
+)
+from packit.constants import (
+    PACKAGE_LONG_OPTION,
+    PACKAGE_OPTION_HELP,
+    PACKAGE_SHORT_OPTION,
+)
+
+logger = logging.getLogger(__name__)
+
+
+@click.command("scan-in-osh", context_settings=get_context_settings())
+@pass_config
+@cover_packit_exception
+@iterate_packages
+@click.option(
+    PACKAGE_SHORT_OPTION,
+    PACKAGE_LONG_OPTION,
+    multiple=True,
+    help=PACKAGE_OPTION_HELP.format(action="build"),
+)
+@click.option(
+    "--target",
+    help="Chroot to build in. (defaults to 'fedora-rawhide-x86_64')",
+    default="fedora-rawhide-x86_64",
+)
+@click.option(
+    "--base-srpm",
+    help="Base SRPM to perform a differential build",
+    default=None,
+)
+@click.option(
+    "--comment",
+    help="Comment for the build",
+    default="Submitted through Packit.",
+)
+@click.argument("path_or_url", type=LocalProjectParameter(), default=os.path.curdir)
+def scan_in_osh(
+    config,
+    path_or_url,
+    package_config,
+    target,
+    base_srpm,
+    comment,
+):
+    """
+    Perform a scan through OpenScanHub
```

Comment:
I would say we could add more detailed info that you are mentioning in the PR description here, e.g. link to the OpenScanHub docs, and mentioning the need for a Kerberos ticket.